sample/replication/docker-compose.yml

```diff
@@ -58,7 +63,7 @@ services:
       # DESTROY_DATABASE_ON_RESTART will default to True if not specified.
       # If specified other than True, it will prevent slave from destroying
       # database on restart
-#      DESTROY_DATABASE_ON_RESTART: 'False'
+      DESTROY_DATABASE_ON_RESTART: 'False'
```

Comment:
Dont we want slaves to always 'start fresh' though @NyakudyaA ?

Reply:
Ah this in int he master section?

Reply:
* The reason for not wanting slaves to start afresh each time is in instances when we have a master that is very big > 6g and fetching the base back up sometimes takes time. 

* So if a user restarts his database then it will just continue from an existing base backup that was already made rather than recreating it each time.

Reply:
Can we get this merged and forward ported to the develop and 10.x branches?